scripts/mkimage

```diff
@@ -255,6 +255,15 @@ elif [ "$BOOTLOADER" = "u-boot" ]; then
     fi
 fi # bootloader
 
+# run device or project specific mkimage
+  if [ -n "$DEVICE" -a -f $PROJECT_DIR/$PROJECT/devices/$DEVICE/mkimage ]; then
+    echo "image: running device mkimage..."
+    . $PROJECT_DIR/$PROJECT/devices/$DEVICE/mkimage
+  elif [ -f $PROJECT_DIR/$PROJECT/mkimage ]; then
+    echo "image: running project mkimage..."
+    . $PROJECT_DIR/$PROJECT/mkimage
```

Comment:
Excellent point, I will move it to `scripts/mkimage`

https://github.com/Kwiboo/LibreELEC.tv/blob/tinkerboard/projects/TinkerBoard/mkimage is the script I used in my TinkerBoard project, I did not want to clutter the main `scripts/mkimage` with even more project/device specific changes